### PR TITLE
Add friendships/show

### PR DESCRIPTION
**Pull request by yyoshiki41**
This change is [Reviewable](https://reviewable.io/reviews/dghubble/go-twitter/48)

Added `friendships/show`.
ref. https://dev.twitter.com/rest/reference/get/friendships/show

thanks!

### CODE REVIEW

File: twitter/friendships.go [hunk 1]
```diff
@@ -53,3 +53,57 @@ func (s *FriendshipService) Destroy(params *FriendshipDestroyParams) (*User, *ht
...
+	ScreenName           string `json:"screen_name"`
+	Following            bool   `json:"following"`
+	FollowedBy           bool   `json:"followed_by"`
+	CanDm                bool   `json:"can_dm"`
+	Blocking             *bool  `json:"blocking"`
+	Muting               *bool  `json:"muting"`
+	AllReplies           *bool  `json:"all_replies"`
+	WantRetweets         *bool  `json:"want_retweets"`
+	MarkedSpam           *bool  `json:"marked_spam"`
+	NotificationsEnabled *bool  `json:"notifications_enabled"`
```
Comment (dghubble) [inline twitter/friendships.go]: some of these are `bool` and others are `*bool`

File: twitter/friendships_test.go [hunk 2]
```diff
@@ -45,3 +45,31 @@ func TestFriendshipService_Destroy(t *testing.T) {
...
+
+func TestFriendshipService_Show(t *testing.T) {
+	httpClient, mux, server := testServer()
+	defer server.Close()
+
+	mux.HandleFunc("/1.1/friendships/show.json", func(w http.ResponseWriter, r *http.Request) {
+		assertMethod(t, "GET", r)
+		assertQuery(t, map[string]string{"source_screen_name": "foo", "target_screen_name": "bar"}, r)
+		w.Header().Set("Content-Type", "application/json")
+		fmt.Fprintf(w, `{ "relationship": { "source": { "can_dm": false, "blocking": null, "muting": null, "id_str": "8649302", "all_replies": null, "want_retweets": null, "id": 8649302, "marked_spam": null, "screen_name": "foo", "following": false, "followed_by": false, "notifications_enabled": null }, "target": { "id_str": "12148", "id": 12148, "screen_name": "bar", "following": false, "followed_by": false } } }`)
```
Comment (dghubble) [inline twitter/friendships_test.go]: Several of these fields aren't actually tested against. You can strip them if they're not a part of the deserialization sanity test.

**Comment by dghubble** [comment]
Added in #52

**Comment by yyoshiki41** [comment]
Sorry for late reply.
Thanks for adding new endpoints https://github.com/dghubble/go-twitter/pull/52 !!

**Pull request closed without merging**